eng/pipelines/templates/steps/resolve-build-platforms.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  - name: ServiceDirectory
+    type: string
+    default: ''
+
+steps:
+    # when we merge pipeline v3, this check will change to examining the targeting string $(TargetingString)
+    # as the generate-pr-diff call + resolution will be present in targeting-string-resolve.yml.
+    # until then, we simply check to see if we're targeting storage service directory
+    - pwsh: |
+        if ("${{ parameters.ServiceDirectory }}" -eq "storage") {
+          Write-Host "Targeting storage, enabling extension build."
+          Write-Host "##vso[task.setvariable variable=ENABLE_EXTENSION_BUILD]true"
```

Comment:
Where is this variable consumed?

Reply:
It's coming! I have a todo checkbox not checked yet. Going to knock it out now. 👍

Reply:
@weshaggard g2g now.!

This is core, note that the windows/mac are shorter:
![image](https://github.com/user-attachments/assets/3202bae5-88ca-422d-aa7b-6b12c04757dd)

And this is storage: